src/main/java/org/dependencytrack/model/VulnerabilityScan.java

```diff
@@ -75,6 +76,21 @@ public enum TargetType {
     @Column(name = "RECEIVED_RESULTS")
     private int receivedResults;
 
+    /**
+     * The threshold percentage value of failure for this scan.
+     */
+    @Persistent
+    @Column(name = "FAILURE_THRESHOLD")
+    private double failureThreshold;
+
+    @Persistent
+    @Column(name = "SCAN_FAILED")
```

Comment:
we should make new fields nullable explicitly. On Prod, we will have data in this table and it will fail on startup if field is not nullable